sdk/services/prices.ts

```diff
@@ -28,40 +28,22 @@ import * as sdkErrors from '../common/errors';
 
 const DEBUG_WS = false;
 const LOG_WS = process.env.NODE_ENV !== 'production' && DEBUG_WS;
+const PRICE_SERVER = process.env.NEXT_PUBLIC_DEFAULT_PRICE_SERVICE === 'KWENTA' ? 'KWENTA' : 'PYTH';
```

Comment:
nitpick but would be good to call this DEFAULT_PRICE_SERVER just so it's distinct from the variable price server

Reply:
Done

Reply:
Yes. That's true. Moved this to the connectToPyth.

Reply:
Thanks. Updated.

Reply:
Thanks for the reminder. I have reverted the changes and missed this one in the redo. Corrected.